packages/cli-repl/src/index.ts

```diff
@@ -13,6 +14,7 @@ export {
   USAGE,
   TELEMETRY,
   CliRepl,
+  redactPwd,
```

Comment:
I think i added this in some PR, not sure what you think about it, but i would not export anything as public unless is necessary. 

If others start to use what we expose we will be limited in the possibility to change things, Is kinda the same thing like marking something that is private as public.